packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -350,6 +350,7 @@
                 >
                     <oui-tile-description>
                         <p
+                            data-ng-click="$ctrl.trackKarapace($event)"
```

Comment:
This will trigger the tracking on click on the whole sentence, which is not what should be done. Also having a click on a non-clickable element, should be avoided.

Reply:
@JeremyDec 

In the function body added an extra check to consider only the clicks on anchor tag.  So tracking would be appropriate.
As per your second point regarding adding a click handler on non-clickable element, I dont see any other better options. The link is part of message which needs translation. Any suggestions on how this can be improved?

Reply:
Yes I've updated my comment in fact.
For the on-click on `<p>` element, the fact here is that the sentence contains the link. Maybe the whole phrase should be the anchor content, in order to wrap it all inside a `<a>`? Or at least be splitted to extract the link from the sentence. It requires functional approval I guess

Reply:
did you try to manually compile your link with an ng-click directive in your controller
```js
$compile('<a href="..." ng-click="$ctrl.trackKarapace">Karapace</a>', $scope);
```
then use the result in translate values ? i'm not sure it'll work as it depends on angular translate sanitization settings but you should give it a try

Reply:
There's also `translate-compile` directive that might help achieve what you want https://angular-translate.github.io/docs/#/api/pascalprecht.translate.directive:translate

Reply:
@JeremyDec  @marie-j  @frenautvh 
Unfortunately, both the above suggestions did not work.